application-forum-ui/src/main/resources/ForumCode/Macros.xml

```diff
@@ -155,9 +155,9 @@
         #set ($commentEditRedirect = $commentDoc.getURL('get', $escapetool.url({'xpage': 'xpart', 'vm': 'conversations.vm'})))
         #set ($editQueryString = $escapetool.url({'number': $comment.number,
                                                   'xredirect': $commentEditRedirect,
-                                                  'xpage': 'xpart',
-                                                  'vm': 'commentsinline.vm'}))
-        #set ($editURL = $commentDoc.getURL('view', $editQueryString))
+                                                  'xpage': 'get',
+                                                  'commentEditForm': 'true'}))
+        #set ($editURL = $commentDoc.getURL('get', $editQueryString))
```

Comment:
Same here. Would be good to explain why you get the comment form through ``AnswerSheet`` rather than ``commentsinline.vm``. And if there is a reason for specifying ``xpage=get`` even though the action is ``get`` you need to mention it because somebody may think it's redundant and remove it thus breaking the feature.

Reply:
@ignatescustefan you still need to add a comment here to explain why ``'xpage': 'get'`` is needed. Maybe because ``commentsinline.vm`` [checks the ``xpage`` request parameter](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm#L389)? I don't remember precisely, but I'm pretty sure it wasn't working correctly without it.